src/kubernetes.h

```diff
@@ -136,10 +136,10 @@ class KubernetesReader {
   json::value GetOwner(const std::string& ns, const json::Object* owner_ref)
       const throw(QueryException, json::Exception);
 
-  // For a given object, returns the top-level owner object.
-  // When there are multiple owner references, follows the first one.
-  json::value FindTopLevelOwner(const std::string& ns, json::value object) const
-      throw(QueryException, json::Exception);
+  // For a given object, returns the top-level controller object.
+  // When there are multiple controller references, follows the first one.
```

Comment:
This is no longer true.

Reply:
done

Reply:
It implicitly is - but yes confusing. Removed the line.